common/src/main/java/net/draycia/carbon/common/messages/CarbonMessageSource.java

```diff
@@ -151,7 +148,16 @@ private void reloadTranslations() throws IOException {
     private void readLocale(final Path localeDirectory, final Path localeFile, final Locale locale) {
         this.logger.info("Found locale {} ({}) in: {}", locale.getDisplayName(), locale, localeFile);
 
-        final Properties properties = new Properties();
+        final Properties properties = new Properties() {
```

Comment:
Would be nice to make this a `private static final class OrderedProperties extends Properties` instead of an anonymous class, but not a big deal, can also just be done when/if this code needs to get re-used somewhere else.

Reply:
I considered implementing an OrderedProperties class, but didn't do it because I didn't think it was necessary 👍